policyengine_us/variables/gov/states/ct/tax/income/ct_amt.py

```diff
@@ -0,0 +1,33 @@
+from policyengine_us.model_api import *
+
+
+class ct_amt(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Connecticut alternative minimum tax"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.CT
+    reference = (
+        "https://portal.ct.gov/-/media/DRS/Forms/2022/Income/2022-CT-1040-Instructions_1222.pdf#page=2"  # line 9
+        "https://portal.ct.gov/-/media/DRS/Forms/2022/Income/CT-1040_1222.pdf#page=1"  # line 9
+        "https://portal.ct.gov/-/media/DRS/Forms/2022/Income/CT-6251_1222.pdf#page=1"  # line 23
+    )
+
+    def formula(tax_unit, period, parameters):
+        # assign amt parameters
```

Comment:
I think the easiest way is to use every line from form https://portal.ct.gov/-/media/DRS/Forms/2022/Income/CT-6251_1222.pdf#page=1 ; it will be a lot clearer, let me know if you have questions for the individual lines

Reply:
We will need to create a file `ct_federal_alternative_minimum_tax` where we include lines 1 -14